<!doctype html> <html lang=en > <meta charset=UTF-8 > <meta name=google-site-verification  content=-EO4jkD2FyQx6dVr0LgXdj1QsnymGzVf5II_GoAOky4 > <meta name=viewport  content="width=device-width, initial-scale=1"> <link rel=stylesheet  href="/libs/katex/katex.min.css"> <link rel=stylesheet  href="/css/franklin.css"> <link rel=stylesheet  href="/css/poole_hyde.css"> <style> html {font-size: 17px;} .franklin-content {position: relative; padding-left: 8%; padding-right: 5%; line-height: 1.35em;} @media (min-width: 940px) { .franklin-content {width: 100%; margin-left: auto; margin-right: auto;} } @media (max-width: 768px) { .franklin-content {padding-left: 6%; padding-right: 6%;} } </style> <link rel=icon  href="/assets/favicon.png"> <title>Research</title> <div class=sidebar > <div class="container sidebar-sticky"> <div class=sidebar-about > <h1><a href="/">Andrea Di Gioacchino</a></h1> <p class=lead >Welcome to my personal webpage!</p> </div> <nav class=sidebar-nav > <a class="sidebar-nav-item " href="/">Home</a> <a class="sidebar-nav-item " href="/pages/CV/">CV</a> <a class="sidebar-nav-item " href="/pages/research/">Research</a> <!-- <a class="sidebar-nav-item {{ispage menu3/*}}active{{end}}" href="/menu_blog/">Blog</a>--> <a class="sidebar-nav-item " href="/pages/contact/">Contact</a> </nav> <!-- <p>&copy; <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA 4.0</a> by {{fill author}}.</p>--> </div> </div> <div class="content container"> <div class=franklin-content > <h1 id=research_interests ><a href="#research_interests" class=header-anchor >Research interests</a></h1> <p>The guiding principle of my research activity is the <strong>interdisciplinarity</strong>: in most of my research efforts, I use ideas and techniques developed in the context of <strong>statistical physics</strong> to model problems coming from different research areas.</p> <p>As a first example, during my PhD I focused on <strong>combinatorial optimization problems</strong>, and in particular how the embedding on problems in low-dimensional Euclidean space &#40;such as in 1 or 2 dimensions&#41; affects the statistical properties of the solutions.</p> <p>Then, during my first year of postdoc, I moved toward biological applications and in particular <strong>computational approaches to molecular biology</strong>. This led me to learn how to build <strong>data-driven models</strong> and how to obtain their parameters with <strong>Bayesian inference</strong> techniques, tools which I consider now a central feature of my profile as a researcher.</p> <p>Successively the main focus of my research shifted towards exploring effects of <strong>virus-host interactions</strong> at the level of their genomes, on topics such as the evolution of the nucleotide usage of viruses after <strong>host shifts</strong>, the determination of nucleotide patterns in viruses recognized by the host <strong>immune system</strong>, and the discovery of <strong>virus-associated molecular patterns</strong> in the human genome.</p> <p>More recently I started working on viruses of bacteria, the so-called bacteriophages or phages, and on machine learning methods to design optimal phages and phage cocktails for <strong>phage therapy</strong>.</p> <h1 id=publications ><a href="#publications" class=header-anchor >Publications</a></h1> <p>Here I list my scientific publications, whether they are actually published in peer-reviewed journals or only available in preprint servers. </p> <p>For each paper I also try to explain briefly what we did and why. </p> <hr /> <ul> <li><p><strong><em>Deciphering the code of viral-host adaptation through maximum entropy models</em></strong>, in collaboration with <em>Benjamin D Greenbaum, Remi Monasson</em>, and <em>Simona Cocco</em>, available in <a href="https://www.biorxiv.org/content/10.1101/2023.10.28.564530v1"><em>bioRxiv</em></a>.</p> <blockquote> <p>In this work, we investigate virus&#39; genetic changes when they jump from one host to another, a critical topic in the study of emerging pathogens. We found that viruses from various families have unique strategies for tuning their nucleotide usage when they infect the same host, and this insight can be leveraged to predict which host a viral sequence came from, even when the sequence is very different from any sequence known. We also demonstrated how this information can be used to design better RNA vaccines and other nucleic acid-based therapies, and to track the evolution of viruses after an host jump.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Designing molecular RNA switches with Restricted Boltzmann machines</em></strong>, in collaboration with <em>Jorge Fernandez-de-Cossio-Diaz, Pierre Hardouin, Francois-Xavier Lyonnet du Moutier, Bertrand Marchand, Yann Ponty, Bruno Sargueil, Remi Monasson</em>, and <em>Simona Cocco</em>, available in <a href="https://www.biorxiv.org/content/10.1101/2023.05.10.540155.abstract"><em>bioRxiv</em></a>.</p> <blockquote> <p>Machine learning tools can be used to discover meaningful patterns in biological sequences, such as RNA molecules, that are related to the molecule functions. In this work we show how to use a specific type of machine learning tool, called Restricted Boltzmann Machines, to design RNA molecules that can switch between two different structures, and we show that the designed molecules are able to perform the desired function in vitro. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Repeats Mimic Pathogen-Associated Patterns Across a Vast Evolutionary Landscape</em></strong>, in collaboration with <em>Petr Šulc, Alexander Solovyov, Sajid A Marhon, Siyu Sun, Håvard T Lindholm, Raymond Chen, Amir Hosseini, Hua Jiang, Bao-Han Ly, Parinaz Mehdipour, Omar Abdel-Wahab, Nicolas Vabret, John LaCava, Daniel D De Carvalho Rémi Monasson, Simona Cocco</em>, and <em>Benjamin D Greenbaum</em>, available in <a href="https://www.biorxiv.org/content/10.1101/2021.11.04.467016v2.abstract"><em>bioRxiv</em></a>.</p> <blockquote> <p>In this work we analyze the human genome using statistical-physics inspired computational tools to detect regions &#40;which in most cases are in the so-called &quot;non-coding genome&quot;&#41; that have viral-like patterns, and that could play a role in interacting with the innate immune system. We also discuss their evolution within and outside the human genome, pointing out cases where possibly immunogenic patterns seems to be conserved during genome evolution or across organisms. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>A transfer-learning approach to predict antigen immunogenicity and T-cell receptor specificity</em></strong>, in collaboration with <em>Barbara Bravi, Andrea Di Gioacchino, Jorge Fernandez-de-Cossio-Diaz, Aleksandra M Walczak, Thierry Mora, Simona Cocco</em>, and <em>Rémi Monasson</em>, published in <a href="https://elifesciences.org/articles/85126"><em>eLife</em></a> on 2023 &#40;open access&#41;.</p> <blockquote> <p>In this work we show how to use an implementation of transfer learning using Restricted Boltzmann Machines to analyze the changes between two datasets, and we apply this idea to predict antigen immunogenicity and T-cell receptor specificity, two well-known open problems in immunology.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Generative and interpretable machine learning for aptamer design and analysis of in vitro sequence selection</em></strong>, in collaboration with <em>Jonah Procyk, Marco Molari, Jonh S Schreck, Yu Zhou, Yan Liu, Rémi Monasson, Simona Cocco</em>, and <em>Petr Šulc</em>, published in <a href="https://journals.plos.org/ploscompbiol/article?id&#61;10.1371/journal.pcbi.1010561"><em>PLoS Computational Biology</em></a> on 2022 &#40;open access&#41;.</p> <blockquote> <p>Aptamers are small DNA or RNA molecules selected in the lab to interact specifically with some relevant protein. For instance <a href="https://en.wikipedia.org/wiki/Anti-thrombin_aptamers">thrombin-binding aptamers</a> have been developed and tested as anti-coagulant agents to be used during surgery. In this work we show how, using machine learning, we are able to better understand the results of the experiments done to obtain new aptamers, and that we can use this added knowledge to design new aptamers with the wanted properties. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>sgDI-tector: defective interfering viral genome bioinformatics for detection of coronavirus subgenomic RNAs</em></strong>, in collaboration with <em>Rachel Legendre, Yannis Rahou, Valérie Najburg, Pierre Charneau, Benjamin D Greenbaum, Frédéric Tangy, Sylvie van der Werf, Simona Cocco</em>, and <em>Anastassia V Komarova</em>, published in <a href="https://rnajournal.cshlp.org/content/28/3/277.short"><em>RNA</em></a> on 2022 &#40;open access&#41;.</p> <blockquote> <p>SARS-CoV-2, as other coronaviruses, has a complex mechanism to translate its genetic code into proteins, that results in many sub-populations of fragments of viral genome &#40;subgenomic RNAs&#41;, which are replicated and translated independently from each other. Is it possible to detect these fragments and to quantify their expression level from the sequencing data coming out of infected cells? In this work, we took a pre-existing bioinformatic tool &#40;DI-tector&#41; and modified it to accomplish this task. We show that the resulting algorithm, sgDI-tector, works as well as other state-of-the-art approaches &#40;and sometimes better&#41;, and we made the software publicly available to help other research groups in detecting subgenomic RNAs.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>The heterogeneous landscape and early evolution of pathogen-associated CpG dinucleotides in SARS-CoV-2</em></strong>, in collaboration with <em>Petr Šulc, Anastassia V Komarova, Benjamin D Greenbaum, Rémi Monasson</em>, and <em>Simona Cocco</em>, published in <a href="https://academic.oup.com/mbe/article/38/6/2428/6130826"><em>Molecular Biology and Evolution</em></a> on 2021 &#40;open access&#41;.</p> <blockquote> <p>Viruses and hosts evolve together, modifying their genetic code and adapting it to the changing environment. This process takes tens to hundreds of years and it is particularly apparent when a virus jumps from an host to another. A well-known example is the usage of cytosines followed by guanines in the genome: humans have a surprising low number of these short motifs &#40;called CpG motifs&#41; in their genome, and can detect as external any genome with too many CpG motifs. For this reason, if a virus jumps into human hosts, it must have a low enough number of CpG motifs, and it will adapt by loosing more and more such motifs. In this work, we discuss this virus-host interaction for SARS-CoV-2, and exploit the incredibly large public database of viral genomes to show that, in genomic regions with high level of CpG motifs, the adaptation process of the virus to the human host had immediately begun. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Perils of embedding for sampling problems</em></strong>, in collaboration with <em>Jeffrey Marshall</em> and <em>Eleanor G Rieffel</em>, published in <a href="https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.2.023020"><em>Physical Review Research</em></a> on 2020 &#40;open access&#41;.</p> <blockquote> <p>Quantum annealer are analog quantum computers &#40;that is, computers exploiting directly quantum phenomena to perform computations&#41; that have been proposed as a tool to solve more efficiently hard combinatorial optimization problem. More recently, they have also been proposed as tools to <em>sample</em> from probability distributions, a known very hard problem in its most general setting. The quantum annealers that we have now, however, need a specific preprocessing called <em>minor embedding</em> to deal with arbitrary problems, that can introduce errors as ambiguities in the sample obtained. While many works have focused on how to mitigate these errors when the objective consists in solving a combinatorial optimization problem, here we focus on the case of sampling. We show that the errors introduced through embedding are expected to be very relevant in the case of sampling, especially for large interesting problems. Moreover, we suggest an empirical procedure involving a post-processing through classical &#40;as opposed to quantum&#41; hardware to mitigate these issues. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Large deviation of the free energy in the p-spin glass spherical model</em></strong>, in collaboration with <em>Mauro Pastore</em> and <em>Pietro Rotondo</em>, published in <a href="https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.1.033116"><em>Physical Review Research</em></a> on 2019 &#40;open access&#41;.</p> <blockquote> <p>In statistical physics of disordered systems the focus is typically on estimating average values of relevant observables of the system for relevant values of the parameters. Here we used the technical tools developed for spin glasses together with the theory of large deviations to describe fluctuations far away from the average values, in the pedagogical case of the spherical p-spin model. In an attempt to better understand the approximations and assumptions usually done in spin glass computations, we also discuss an alternative view of some technical problems suggesting that a replica symmetry breaking scheme is at play. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Selberg integrals in 1D random Euclidean optimization problems</em></strong>, in collaboration with <em>Sergio Caracciolo, Enrico M Malatesta</em>, and <em>Luca G Molinari</em>, published in <a href="https://iopscience.iop.org/article/10.1088/1742-5468/ab11d7"><em>Journal of Statistical Mechanics: Theory and Experiment</em></a> on 2019 &#40;preprint available in <a href="https://arxiv.org/abs/1810.00587">arXiv</a>&#41;.</p> <blockquote> <p>Selberg integrals are a famous family of integrals introduced by <a href="https://en.wikipedia.org/wiki/Atle_Selberg">Atle Selberg</a>. In this quite technical paper, we show how to use an extension of these integrals to compute the cost of some euclidean combinatorial optimization problems, such has the assignment, in one spatial dimension, for any number of points.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Average optimal cost for the Euclidean TSP in one dimension</em></strong>, in collaboration with <em>Sergio Caracciolo, Enrico M Malatesta</em>, and <em>Carlo Vanoni</em>, published in <a href="https://iopscience.iop.org/article/10.1088/1751-8121/ab1600"><em>Journal of Physics A: Mathematical and Theoretical</em></a> on 2019 &#40;preprint available in <a href="https://arxiv.org/abs/1811.08265">arXiv</a>&#41;.</p> <blockquote> <p>In this work we consider the classic version of the Traveling Salesman Problem &#40;different from the <em>bipartite</em> version that we considered in previous papers&#41;, and we manage to describe the geometry of the solution when the points are uniformly chosen at random on a line. Although this version of the problem is in general more difficult to deal to with our methods, we still have been able to obtain the average cost of the solution in the limit of large number of points.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Exact value for the average optimal cost of the bipartite traveling salesman and two-factor problems in two dimensions</em></strong>, in collaboration with <em>Riccardo Capelli, Sergio Caracciolo</em>, and <em>Enrico M Malatesta</em>, published in <a href="https://journals.aps.org/pre/abstract/10.1103/PhysRevE.98.030101"><em>Physical Review E</em></a> on 2018 &#40;preprint available in <a href="https://arxiv.org/abs/1807.03559">arXiv</a>&#41;.</p> <blockquote> <p>Solving the Traveling Salesman Problem in two dimensions means to be able to tell, for any possible position of <span class=katex ><span class=katex-mathml ><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>N</mi></mrow><annotation encoding="application/x-tex">N</annotation></semantics></math></span><span class=katex-html  aria-hidden=true ><span class=base ><span class=strut  style="height:0.6833em;"></span><span class="mord mathnormal" style="margin-right:0.10903em;">N</span></span></span></span> cities on a &#40;flat&#41; map, the shortest path traveling to all of them and coming back to the starting point. This is hard, and indeed solving this problem would allow anybody to get a reward of a million dollars from the Clay Mathematics Institute &#40;more info <a href="https://www.claymath.org/millennium-problems/p-vs-np-problem">here</a>&#41;. In this paper, we consider a variant of this problem &#40;which is at least as hard as the problem itself&#41;, and showed that, for large number of cities, we can compute the length of the optimal tour by solving a much simpler problem &#40;and no, this is not enough to win the prize&#33;&#41;.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Plastic number and possible optimal solutions for an Euclidean 2-matching in one dimension</em></strong>, in collaboration with <em>Sergio Caracciolo</em> and <em>Enrico M Malatesta</em>, published in <a href="https://iopscience.iop.org/article/10.1088/1742-5468/aad3f7"><em>Journal of Statistical Mechanics: Theory and Experiment</em></a> on 2018 &#40;preprint available in <a href="https://arxiv.org/abs/1805.07178">arXiv</a>&#41;.</p> <blockquote> <p>In this work we consider a combinatorial optimization problem known as the 2-matching. It consists, given a set of points, in joining them in loops &#40;with more than 2 points&#41; so that the total length &#40;or, better, cost, which is a function of the length&#41; of the loops is the smallest possible. This problem is somehow intermediate between the matching problem &#40;where we have segments connecting points instead of loops&#41; and the traveling salesman problem &#40;where the loop has to be a single one&#41;. We showed that the solution, when the points are on a line, is composed by loops involving a little number of points, and depending on the specific points positions it can be any of a large number of solution &#40;the so-called <em>plastic number</em> to the number of points&#41;. Even if we could not characterize exactly the solution for general point positions, we have been able to compute the average cost of the solution when the number of points goes to infinity.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Solution for a bipartite Euclidean traveling-salesman problem in one dimension</em></strong>, in collaboration with <em>Sergio Caracciolo, Marco Gherardi</em>, and <em>Enrico M Malatesta</em>, published in <a href="https://journals.aps.org/pre/abstract/10.1103/PhysRevE.97.052109"><em>Physical Review E</em></a> on 2018 &#40;preprint available in <a href="https://arxiv.org/abs/1802.01545">arXiv</a>&#41;.</p> <blockquote> <p>Euclidean combinatorial optimization problems are everywhere: for instance, they are routinely solved by Google Maps to find shortest routes to go to work, or to a party. Here we considered the problem of finding a cycle passing through a set of points on a line so that a given cost, which depends on the points&#39; distances, is minimized. To do so, we proved some geometrical properties of the optimal solution for arbitrary points&#39; positions, and managed to use these properties to compute the average cost of the solution when points are randomly placed on a line.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Unified Fock space representation of fractional quantum Hall states</em></strong>, in collaboration with <em>Luca G Molinari, Vittorio Erba</em>, and <em>Pietro Rotondo</em>, published in <a href="https://journals.aps.org/prb/abstract/10.1103/PhysRevB.95.245123"><em>Physical Review B</em></a> on 2017 &#40;preprint available in <a href="https://arxiv.org/abs/1705.07073">arXiv</a>&#41;.</p> <blockquote> <p>The fractional quantum Hall effect &#40;FQHE&#41; is a physical phenomenon described in 1982, which is still the topic of active research in present days. Here we contribute to this line of research, showing how to use a single framework to describe two distinct cases of the simplest FQHE theory &#40;the <em>bosonic</em> and <em>fermionic</em> Laughlin wave functions&#41;, and elucidating where &#40;mathematically speaking&#41; the differences in these two cases come from.</p> </blockquote> </ul> <h1 id=conferences ><a href="#conferences" class=header-anchor >Conferences</a></h1> <p>I co-organized a conference in Paris about Innate and Adaptive Recognition of Antigens and NeoAntigens, more info at the <a href="https://sites.google.com/view/qbio-iarana/home-page">conference website</a>.</p> <div class=page-foot > <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA 4.0</a> Andrea Di Gioacchino. Last modified: November 05, 2023. Website built with <a href="https://github.com/tlienart/Franklin.jl">Franklin.jl</a> and the <a href="https://julialang.org">Julia programming language</a>. </div> </div> </div>