<!doctype html> <html lang=en > <meta charset=UTF-8 > <meta name=viewport  content="width=device-width, initial-scale=1"> <link rel=stylesheet  href="/css/franklin.css"> <link rel=stylesheet  href="/css/poole_hyde.css"> <style> html {font-size: 17px;} .franklin-content {position: relative; padding-left: 8%; padding-right: 5%; line-height: 1.35em;} @media (min-width: 940px) { .franklin-content {width: 100%; margin-left: auto; margin-right: auto;} } @media (max-width: 768px) { .franklin-content {padding-left: 6%; padding-right: 6%;} } </style> <link rel=icon  href="/assets/favicon.png"> <title>Research</title> <div class=sidebar > <div class="container sidebar-sticky"> <div class=sidebar-about > <h1><a href="/">Andrea Di Gioacchino</a></h1> <p class=lead >Welcome to my personal webpage!</p> </div> <nav class=sidebar-nav > <a class="sidebar-nav-item " href="/">Home</a> <a class="sidebar-nav-item " href="/pages/CV/">CV</a> <a class="sidebar-nav-item " href="/pages/research/">Research</a> <!-- <a class="sidebar-nav-item {{ispage menu3/*}}active{{end}}" href="/menu_blog/">Blog</a>--> <a class="sidebar-nav-item " href="/pages/contact/">Contact</a> </nav> <!-- <p>&copy; <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA 4.0</a> by {{fill author}}.</p>--> </div> </div> <div class="content container"> <div class=franklin-content > <h1 id=research_interests ><a href="#research_interests" class=header-anchor >Research interests</a></h1> <h1 id=publications ><a href="#publications" class=header-anchor >Publications</a></h1> <p>Here I list my publications in peer-reviewed journals. </p> <p>For each paper I also try to explain what we did and why &#40;aiming at non-expert audience&#41;, without trying to sell anything to anyone. ;&#41; </p> <hr /> <ul> <li><p><strong><em>sgDI-tector: defective interfering viral genome bioinformatics for detection of coronavirus subgenomic RNAs</em></strong>, in collaboration with <em>Rachel Legendre, Yannis Rahou, Valérie Najburg, Pierre Charneau, Benjamin D Greenbaum, Frédéric Tangy, Sylvie van der Werf, Simona Cocco</em>, and <em>Anastassia V Komarova</em>, published in <a href="https://rnajournal.cshlp.org/content/28/3/277.short"><em>RNA</em></a> on 2022 &#40;open access&#41;.</p> <blockquote> <p>SARS-CoV-2, as other coronaviruses, has a complex mechanism to translate its genetic code into proteins, that results in many sub-populations of fragments of viral genome &#40;subgenomic RNAs&#41;, which are replicated and translated independently from each other. Is it possible to detect these fragments and to quantify their expression level from the sequencing data coming out of infected cells? In this work, we took a pre-exisiting bioinformatic tool &#40;DI-tector&#41; and modified it to accomplish this task. We show that the resulting algorithm, sgDI-tector, works as well as other state-of-the-art approaches &#40;and sometimes better&#41;, and we made the software publicly available to help other research groups in detecting subgenomic RNAs.</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>The heterogeneous landscape and early evolution of pathogen-associated CpG dinucleotides in SARS-CoV-2</em></strong>, in collaboration with <em>Petr Sulc, Anastassia V Komarova, Benjamin D Greenbaum, Rémi Monasson</em>, and <em>Simona Cocco</em>, published in <a href="https://academic.oup.com/mbe/article/38/6/2428/6130826"><em>Molecular Biology and Evolution</em></a> on 2021 &#40;open access&#41;.</p> <blockquote> <p>Viruses and hosts evolve together, modifying their genetic code and adapting it to the changing environment. This process takes tens to hundreds of years and it is particularly apparent when a virus jumps from an host to another. A well-known example is the usage of cytosines followed by guanines in the genome: humans have a surprising low number of these short motifs &#40;called CpG motifs&#41; in their genome, and can detect as external any genome with too many CpG motifs. For this reason, if a virus jumps into human hosts, it must have a low enough number of CpG motifs, and it will adapt by loosing more and more such motifs. In this work, we discuss this virus-host interaction for SARS-CoV-2, and exploit the incredibly large public database of viral genomes to show that, in genomic regions with high level of CpG motifs, the adaptation process of the virus to the human host had immediately begun. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Perils of embedding for sampling problems</em></strong>, in collaboration with <em>Jeffrey Marshall</em> and <em>Eleanor G Rieffel</em>, published in <a href="https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.2.023020"><em>Physical Review Research</em></a> on 2020 &#40;preprint available in <a href="http://128.84.21.203/abs/1909.12184">arXiv</a>&#41;.</p> <blockquote> <p>description</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Large deviation of the free energy in the p-spin glass spherical model</em></strong>, in collaboration with <em>Mauro Pastore</em> and <em>Pietro Rotondo</em>, published in <a href="https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.1.033116"><em>Physical Review Research</em></a> on 2019 &#40;preprint available in <a href="https://arxiv.org/abs/1909.06196">arXiv</a>&#41;.</p> <blockquote> <p>In statistical physics of disordered systems the focus is typically on estimating average values of relevant observables of the system for relevant values of the parameters. Here we used the technical tools developed for spin glasses together with the theory of large deviations to describe fluctuations far away from the average values, in the pedagogical case of the spherical p-spin model. In an attempt to better understand the approximations and assumptions usually done in spin glass computations, we also discuss an alternative view of some technical problems suggesting that a replica symmetry breaking scheme is at play. </p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Selberg integrals in 1D random Euclidean optimization problems</em></strong>, in collaboration with <em>Sergio Caracciolo, Enrico M Malatesta</em>, and <em>Luca G Molinari</em>, published in <a href="https://iopscience.iop.org/article/10.1088/1742-5468/ab11d7"><em>Journal of Statistical Mechanics: Theory and Experiment</em></a> on 2019 &#40;preprint available in <a href="https://arxiv.org/abs/1810.00587">arXiv</a>&#41;.</p> <blockquote> <p>description</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Average optimal cost for the Euclidean TSP in one dimension</em></strong>, in collaboration with <em>Sergio Caracciolo, Enrico M Malatesta</em>, and <em>Carlo Vanoni</em>, published in <a href="https://iopscience.iop.org/article/10.1088/1751-8121/ab1600"><em>Journal of Physics A: Mathematical and Theoretical</em></a> on 2019 &#40;preprint available in <a href="https://arxiv.org/abs/1811.08265">arXiv</a>&#41;.</p> <blockquote> <p>description</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Exact value for the average optimal cost of the bipartite traveling salesman and two-factor problems in two dimensions</em></strong>, in collaboration with <em>Riccardo Capelli, Sergio Caracciolo</em>, and <em>Enrico M Malatesta</em>, published in <a href="https://journals.aps.org/pre/abstract/10.1103/PhysRevE.98.030101"><em>Physical Review E</em></a> on 2018 &#40;preprint available in <a href="https://arxiv.org/abs/1807.03559">arXiv</a>&#41;.</p> <blockquote> <p>description</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Plastic number and possible optimal solutions for an Euclidean 2-matching in one dimension</em></strong>, in collaboration with <em>Sergio Caracciolo</em> and <em>Enrico M Malatesta</em>, published in <a href="https://iopscience.iop.org/article/10.1088/1742-5468/aad3f7"><em>Journal of Statistical Mechanics: Theory and Experiment</em></a> on 2018 &#40;preprint available in <a href="https://arxiv.org/abs/1805.07178">arXiv</a>&#41;.</p> <blockquote> <p>description</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Solution for a bipartite Euclidean traveling-salesman problem in one dimension</em></strong>, in collaboration with <em>Sergio Caracciolo, Marco Gherardi</em>, and <em>Enrico M Malatesta</em>, published in <a href="https://journals.aps.org/pre/abstract/10.1103/PhysRevE.97.052109"><em>Physical Review E</em></a> on 2018 &#40;preprint available in <a href="https://arxiv.org/abs/1802.01545">arXiv</a>&#41;.</p> <blockquote> <p>description</p> </blockquote> </ul> <hr /> <ul> <li><p><strong><em>Unified Fock space representation of fractional quantum Hall states</em></strong>, in collaboration with <em>Luca G Molinari, Vittorio Erba</em>, and <em>Pietro Rotondo</em>, published in <a href="https://journals.aps.org/prb/abstract/10.1103/PhysRevB.95.245123"><em>Physical Review B</em></a> on 2017 &#40;preprint available in <a href="https://arxiv.org/abs/1705.07073">arXiv</a>&#41;.</p> <blockquote> <p>description</p> </blockquote> </ul> <div class=page-foot > <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA 4.0</a> Andrea Di Gioacchino. Last modified: September 21, 2022. Website built with <a href="https://github.com/tlienart/Franklin.jl">Franklin.jl</a> and the <a href="https://julialang.org">Julia programming language</a>. </div> </div> </div>